src/components/common/PeopleAndRoles/CurrentDirectors.vue

```diff
@@ -48,13 +48,12 @@
 
             <!-- Delivery Address -->
             <td class="px-0">
-              <p
-                v-if="IsSame(orgPerson.mailingAddress, orgPerson.deliveryAddress, ['id'])"
-                class="director-detail"
-              >
```

Comment:
This paragraph element added some unwanted styling so I changed it to a span, same as line 62 below.